code/utils/xcmp/src/main.rs

```diff
@@ -111,18 +46,90 @@ pub async fn main() {
 		Command::TransferNative(command) => {
 			transfer_native_asset(command).await;
 		},
+		Command::Sudo(command) => match command.command {
+			SudoCommand::Execute(execute) =>
+				execute_sudo(execute.ask, execute.call, execute.network, execute.suri, execute.rpc)
+					.await,
+			_ => todo!("implement"),
+		},
+	}
+}
+
+macro_rules! decode_call {
```

Comment:
Could these macros just be functions? `dali` is the only network ever passed to them, and `decode_call!` seems to be hard-coded to dali_runtime anyways?

Reply:
mistake. nees to set picasso.